sunspear/backends/database/db.py

```diff
@@ -288,14 +290,23 @@ def activity_get(self,
 
             # audience_activities_query = sql.select(['*']).where(self.activities_table.c.id.in_(audience_query))
             # result_proxy = self.engine.execute(audience_activities_query)
+        """
```

Comment:
random fact: note how triple quotes don't really comment it out, but just make it a string, i.e. a statement that doesn't do anything. except that python concatenates consecutive strings, i.e. this becomes part of the above assertion failure message

Reply:
I do know that ;) I was just being lazy and wanted the code not to execute

Reply:
do you know about `Ctrl+/`? 😊

Reply:
no

Reply:
you can select a block of code and it'll comment it out/in for you!